src/oc_erchef/apps/oc_chef_wm/src/chef_wm_named_sandbox.erl

```diff
@@ -91,11 +91,12 @@ resource_exists(Req, #base_state{chef_db_context = DbContext,
 %% json_class field since even Chef 0.10.z clients do not operate on the returned data.
 from_json(Req, #base_state{reqid = ReqId,
                            chef_db_context = DbContext,
-                           resource_state = #sandbox_state{chef_sandbox = Sandbox}} = State) ->
+                           requestor_id = RequestorId,
+                           resource_state = #sandbox_state{chef_sandbox = Sandbox, sandbox_authz_id = SBAuthz}} = State) ->
```

Comment:
It seems odd to me that we don't have the authz id in the `#sandbox{}` record itself, like we do for other objects -- even though we include it in the data we write to the DB. Have you been able to confirm that the authz id is available in the `#sandbox_state{}` record at this point in time?